### DISCUSSION TITLE
2.6.17 Chinese garbled characters

### DISCUSSION THREAD

**Discussion by CLAY-ZQ** [SCADA-LTS/Scada-LTS#2420] [General]
Hello, I have a question. (common.alarmLevel.info) How to resolve garbled characters in Chinese on the browser interface?

**Comment by Limraj** [comment]
Hello @CLAY-ZQ,
Show this problem on the screen. :)

**Comment by CLAY-ZQ** [comment]
![1](https://user-images.githubusercontent.com/70440817/206080872-f83258a4-b181-4010-a047-d5cf524b1101.png)

![2](https://user-images.githubusercontent.com/70440817/206080889-b918e1ca-a042-4107-bcc9-67f759285206.png)

![3](https://user-images.githubusercontent.com/70440817/206080900-b0381393-35df-46ca-893b-816dfa646423.png)

**Comment by Limraj** [comment]
Hello @CLAY-ZQ,
We will fix this in version 2.7.4.1, where there is no translation in Chinese, there will be an English translation. By the way, we will also improve other translations.

![china](https://user-images.githubusercontent.com/35842300/208004738-e92f3b88-9882-4778-8d88-0d8412b06fc6.png)
![china2](https://user-images.githubusercontent.com/35842300/208004952-ce8f56e1-c6fa-464b-a8eb-52e3f2d2caa5.png)
